pytest.ini

```diff
@@ -7,6 +7,14 @@ markers =
     polarion: Store polarion test ID
     jira: Store jira bug ID
 
+    # Distributions and supported downstream markers
+    upstream: Mark tests as ones suitable to run only upstream.
```

Comment:
How would we handle when a feature behavior changes between releases and when test step changes between releases?

Reply:
how about adding the option to add an indicator like
`up_to_rhoai_2_16` and there will be another test with the new functionality which will have `rhoai_2_17` ?

Reply:
This would add duplicate tests and the repository can potentially get messy and hard to maintain. I don't like this idea.